solutions/silver/cf-702C.mdx

```diff
@@ -319,10 +359,9 @@ left of the city. Define $\texttt{dist}$ to be the minimum of the distance to
 the tower on the right and the distance to the tower on the left. Then, set $r$
 to be the maximum of itself and $\texttt{dist}$.
 
-**Time Complexity:** $\mathcal{O}(N\log N)$
+**Time Complexity:** $\mathcal{O}((N + M)\log M)$
```

Comment:
wouldn't this be n log m?

where's the n+m come from (input? - n, m share the same bound right)

Reply:
how

Reply:
where'd the n + m come from (input?)
bottle neck is in the for loop right 🤔

Reply:
my bad I was ignoring the dominancy. thanks

Reply:
> where'd the n + m come from (input?) bottle neck is in the for loop right 🤔

yes